proteinworkshop/datasets/pdb_dataset.py

```diff
@@ -109,13 +114,20 @@ def create_dataset(self):
             pdb_manager.remove_unavailable_pdbs(update=True)
             log.info(f"{len(pdb_manager.df)} chains remaining")
 
-        log.info(f"Splitting dataset into {self.split_sizes}...")
-        split_names = ["train", "val", "test"]
-        splits = pdb_manager.split_df_proportionally(
-            df=pdb_manager.df,
-            splits=split_names,
-            split_ratios=self.split_sizes,
-        )
+        if self.split_type == "random":
+            log.info(f"Splitting dataset via random split into {self.split_ratios}...")
+            splits = pdb_manager.split_df_proportionally(
+                df=pdb_manager.df,
+                splits=self.splits,
+                split_ratios=self.split_ratios,
+            )
+        
+        elif self.split_type == "sequence_similarity":
+            log.info(f"Splitting dataset via sequence-similarity split into {self.split_ratios}...")
+            log.info(f"Using {self.split_sequence_similarity} sequence similarity for split")
+            pdb_manager.cluster(min_seq_id=self.split_sequence_similarity, update=True, overwrite=True)
```

Comment:
What are your thoughts on having `overwrite=True` by default. Seems like something we'd like to control? Will the split be deterministic?

Reply:
good observation; tried it out and splits are not deterministic. Changed it to be False by default now

Reply:
the case of different seq id levels is already covered so that is still possible

Reply:
I think it makes sense to add an `overwrite` arg to the datamodule which is passed down to the `create_dataset` call. A few datamodules already have this functionality.

Reply:
done, lmk what you think

Reply:
Nice! LGTM